lib/controller/controller.go

```diff
@@ -164,6 +164,14 @@ type ProvisionController struct {
 	// Map of failed claims
 	failedClaimsStats      map[types.UID]int
```

Comment:
We should rename these variables to be consistent, don't need to do it now though.